autosklearn/data/validation.py

```diff
@@ -110,6 +112,23 @@ def validate_target(
         Wrapper around sklearn check_array. Translates a pandas
         Dataframe to a valid input for sklearn.
         """
+
+        # Target data as sparse is not supported
+        if scipy.sparse.issparse(y):
+            raise ValueError("Unsupported target data provided"
+                             "Input targets to auto-sklearn must not be of "
+                             "type sparse. Please convert the target input (y) "
+                             "to a dense array via scipy.sparse.csr_matrix.todense(). "
+                             )
+
+        # No Nan is supported
+        if np.any(pd.isnull(y)):
+            raise ValueError("Target values cannot contain missing/NaN values. "
+                             "You can pre-process your data via: "
```

Comment:
Please drop the statement about imputing targets. This needs techniques from semi-supervised learning which are beyond what Auto-sklearn can do.

Reply:
(and what scikit-learn offers)

Reply:
Done in next push!